test/network_analysis/network_properties.jl

```diff
@@ -127,3 +124,14 @@ cls = conservationlaws(rn3)
 #     end
 #     println("-----------")
 # end
+
+
+### Tests additional network properties ###
+rn4 = @reaction_network begin
```

Comment:
These are really API tests. Aren't these covered there? (I'd at least put them in there instead, within a new `let` block.

Reply:
I was unsure where to put them, but in the end did them here because the functions was not in the API file (but the rs file). Happy to move it.

Reply:
Ahh, then put them in `reactionsystem.jl` I think. That is where the basic `reactionsystem` functionality should go.

Reply:
sorry, mixed up what was were. No, this should be in the API file as you initially said.
(it s the thing in your next comment that is reactionsystem file stuff)